stats/gRPC.md

```diff
@@ -59,13 +64,17 @@ https://github.com/grpc/grpc/blob/master/doc/statuscodes.md
 
 The following set of views are considered minimum required to monitor client-side performance:
 
-| View name                             | Measure                               | Aggregation  | Tags                                   |
-|---------------------------------------|---------------------------------------|--------------|----------------------------------------|
-| grpc.io/client/sent_bytes_per_rpc     | grpc.io/client/sent_bytes_per_rpc     | distribution | grpc_client_method                     |
-| grpc.io/client/received_bytes_per_rpc | grpc.io/client/received_bytes_per_rpc | distribution | grpc_client_method                     |
-| grpc.io/client/roundtrip_latency      | grpc.io/client/roundtrip_latency      | distribution | grpc_client_method                     |
-| grpc.io/client/completed_rpcs         | grpc.io/client/roundtrip_latency      | count        | grpc_client_method, grpc_client_status |
-| grpc.io/client/started_rpcs           | grpc.io/client/started_rpcs           | count        | grpc_client_method                     |
+| View name                                 | Measure                                   | Aggregation  | Tags                                   |
+|-------------------------------------------|-------------------------------------------|--------------|----------------------------------------|
+| grpc.io/client/sent_bytes_per_rpc         | grpc.io/client/sent_bytes_per_rpc         | distribution | grpc_client_method                     |
+| grpc.io/client/received_bytes_per_rpc     | grpc.io/client/received_bytes_per_rpc     | distribution | grpc_client_method                     |
+| grpc.io/client/roundtrip_latency          | grpc.io/client/roundtrip_latency          | distribution | grpc_client_method                     |
+| grpc.io/client/completed_rpcs             | grpc.io/client/roundtrip_latency          | count        | grpc_client_method, grpc_client_status |
+| grpc.io/client/started_rpcs               | grpc.io/client/started_rpcs               | count        | grpc_client_method                     |
+| grpc.io/client/sent_messages_per_method      | grpc.io/client/sent_messages_per_method      | count | grpc_client_method                     |
```

Comment:
I don't think this are necessary in the default views. I would put them in the extra views for the moment. We probably want to have these measurements not enabled by default to avoid unnecessary overhead for unary calls (probably the majority of the calls).

Reply:
Yes I agree. Made them optional for the moment, and add an FAQ about these views.